plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-overview.spec.js

```diff
@@ -72,19 +72,23 @@ test.describe( 'Analytics pages', () => {
 			await page.goto(
 				'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview'
 			);
-			// check the top section (Performance)
-			await page.click(
-				'//button[@title="Choose which analytics to display and the section name"]'
-			);
+			// check the top section
+			await page
+				.locator( 'button.woocommerce-ellipsis-menu__toggle' )
+				.first()
+				.click();
 			await expect( page.locator( 'text=Move up' ) ).not.toBeVisible();
 			await expect( page.locator( 'text=Move down' ) ).toBeVisible();
+			await page.keyboard.press( 'Escape' );
 
-			// check the bottom section (Leaderboards)
-			await page.click(
-				'//button[@title="Choose which leaderboards to display and other settings"]'
-			);
+			// check the bottom section
+			await page
+				.locator( 'button.woocommerce-ellipsis-menu__toggle' )
+				.last()
+				.click();
 			await expect( page.locator( 'text=Move down' ) ).not.toBeVisible();
 			await expect( page.locator( 'text=Move up' ) ).toBeVisible();
+			await page.keyboard.press( 'Escape' );
```

Comment:
Same reasoning cited [here](https://github.com/woocommerce/woocommerce/pull/35492/files#r1013829528) and [here](https://github.com/woocommerce/woocommerce/pull/35492/files#r1013830705).